### ISSUE TITLE
example for optimization

### ISSUE THREAD

**Issue by HajimeKawahara** [HajimeKawahara/exojax#72]
So far, we focus on HMC-NUTS as an application of exojax. But, we can use exojax for gradient-based optimization.

**Comment by HajimeKawahara** [comment]
optax

- chex does not work for jaxlib 0.1.60
- it works for jaxlib 0.1.64

**Comment by HajimeKawahara** [comment]
For the optimization of the Voigt profile, see here:
http://secondearths.sakura.ne.jp/exojax/tutorials/optimize_voigt.html

available since v0.5.5

**Comment by HajimeKawahara** [comment]
ExoJAX should be compatible to [jaxopt](https://github.com/google/jaxopt).

**Comment by HajimeKawahara** [comment]
[JAXopt](https://github.com/google/jaxopt) works for ExoJAX emission spectrum anyway. The following is an example of the fitting using [jaxopt.GradientDescent](https://jaxopt.github.io/stable/_autosummary/jaxopt.GradientDescent.html). 
![gradient_descent_jaxopt](https://user-images.githubusercontent.com/15956904/153967613-bcdf0cd4-6b75-40a7-8219-4a0b01231355.png)

**Comment by HajimeKawahara** [comment]
some parameters (e.g. rotational braodening of vsini) are unstable for optimization, yielding nan quickly. We need to find how to avoid such nan.

**Comment by HajimeKawahara** [comment]
#211 solved the vsini issue. The fitting by JAXopt.GradientDescent looks good.
![gradient_descent_jaxopt](https://user-images.githubusercontent.com/15956904/154021102-feb9b1d8-2c0e-4e5c-abde-ca8b4bf2fea3.png)

**Comment by HajimeKawahara** [comment]
from a farther point
https://user-images.githubusercontent.com/15956904/154577407-ffa61525-1462-4285-b93c-7347f16a7790.mp4
